terraso_backend/apps/shared_data/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+
+from apps.core.models import Group, SlugModel, User
+from apps.shared_data import permission_rules as perm_rules
+
+
+class DataEntry(SlugModel):
+    """
+    Data Entry stores information about resources (usually files) that contain
+    different kind of data used by Landscape managers. Common resource types are
+    csv, xls and JSON files.
+
+    A Data Entry can point to internal or external resources. An internal
+    resource is stored on Terraso's infrastructure and an external resource is
+    stored out of the Terraso's infrastructure. In both cases, the Data Entry
+    only has the URL for that resource as a link to it.
+
+    Attributes
+    ----------
+    name: str
+        any user given name for that resource
+    description: str
+        a longer description explaining the resource
+    resource_type: str
+        the 'technical' type of the resource, usually the mime type
```

Comment:
question: do we have a map of MIME types to file extensions?

Reply:
Python has, and it's going to be used when receiving the file: https://docs.python.org/3/library/mimetypes.html